### DISCUSSION TITLE
最新版本是不是加入了新功能

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#497] [Q&A 问答]
仅仅是吹水，对我没有任何影响和不便。之前一直用这个脚本，https://github.com/Natural-Harmonia-Gropius/InputEvent ，左键长按加速，新版本长按时不能移动，移动就回到原来倍速。右键减速可以移动，跟原来一样，不知道是不是要加入其他新的功能，比如支持触摸啥的，有点期待。或者只是设置问题？

**Comment by natural-harmonia-gropius** [comment]
mpv某个版本加了canceled，移动窗口和关闭内置右键菜单会触发。左键移动的时候是不是窗口也动了，而右键移动时窗口没动

**Reply by sqq135** [reply]
窗口倒是没动。我有点怀疑是不是要支持触摸屏了。安卓的reex用mpv的内核，我就觉得做的不错。

**Reply by hooke007** [reply]
https://github.com/mpv-player/mpv/pull/14003 这个pr支持的多点触控，但是和安卓应该没啥关系。
安卓第三方的libmpv的所有交互都是自己实现的，mpv只用来输出画面而已

![image](https://github.com/user-attachments/assets/0e38bc5a-5ff9-42fe-a8ea-b13a4f41902f)

**Reply by sqq135** [reply]
正好是4个月前，长按的变化还真有可能是这个原因，虽然也没啥影响就是了。

安卓那个我就是提一嘴，几年前就有了，和截图这个不相关。安卓APP真的把门槛下放的非常低了，而且还做的不错。

**Reply by hooke007** [reply]
播放器的前端要是还有门槛，那还叫什么前端。。。

**Reply by sqq135** [reply]
我记得lazy一开始auto_load是要自己安装吧（现版本好像也得改一下），还有些设置需要查教程，也没有跟平常APP一样的菜单。花点时间折腾lazy我觉得挺有趣，但也不适合只想拿来就用的人。（lazy应该算是前端吧）

我说的安卓端的mpv是真的和其他播放器一样，完全不需要教程，也有一些mpv高自由度特点。

（lazy原conf调用核显，色彩很奇怪，我调整之后才好，我也默认了用lazy之前要学一下，可能这对有基础的人来说很简单吧[捂脸]）

**Reply by hooke007** [reply]
lazy不是前端

**Reply by hooke007** [reply]
你可以看一下隔壁vlc的开发模式，官方前后端同时开发。你难道会去用后端的vlc吗，根本不会，前端做的够大而全了，甚至功能全到第三方无法超越。
